arangod/Aql/RestAqlHandler.cpp

```diff
@@ -715,7 +715,6 @@ RestStatus RestAqlHandler::handleUseQuery(std::string const& operation,
         return RestStatus::WAITING;
       }
     }
-    // Used in 3.4.0 onwards.
     answerBuilder.add("done", VPackValue(state == ExecutionState::DONE));
```

Comment:
same here

Reply:
I agree this should be changed, but this is not related to this PR, so I would like to not include it here.